flink-connectors/flink-connector-kafka-0.10/src/main/java/org/apache/flink/streaming/connectors/kafka/internal/KafkaConsumerCallBridge010.java

```diff
@@ -48,4 +59,18 @@ public void seekPartitionToBeginning(KafkaConsumer<?, ?> consumer, TopicPartitio
 	public void seekPartitionToEnd(KafkaConsumer<?, ?> consumer, TopicPartition partition) {
 		consumer.seekToEnd(Collections.singletonList(partition));
 	}
+
+	@Override
+	public void seekPartitionToDate(KafkaConsumer<?, ?> consumer, TopicPartition partition) {
```

Comment:
But from here I can understand why.

Ideally, this method signature should really be `seekPartitionToDate(KafkaConsumer, TopicParitition, Date)`, but that would require the startup date to be passed all the way to the `KafkaConsumerThread`.
This also leads to the fact, which isn't nice, that the `KafkaConsumerThread` lives within the Kafka 0.9 module, while 0.9 doesn't support timestamp-based offsets ...